examples/ads.amp.html

```diff
@@ -265,6 +266,12 @@
       data-host="talos.adman.gr">
   </amp-ad>
 
+  <h2>AdmanMedia</h2>
+  <amp-ad width="300" height="250"
```

Comment:
Cool I am able to see it now. After this Pull Request is merged it usually takes two weeks to hit prod. It's usually towards the end of a week, but not always on an exact data. Is there a concern why you can't setup the header a bit earlier?